src/clients/telegram/index.ts

```diff
@@ -0,0 +1 @@
+export { TelegramClient } from './telegram'
```

Comment:
let's follow the current standard. this way we don't need to manually update this file when **telegram.ts** grows with new exports.

```suggestion
export * from './telegram'
```